{"Modified":"2019-05-01T19:29:00","Published":"2019-05-01T18:29:00","access":{"authentication":"NONE","complexity":"MEDIUM","vector":"NETWORK"},"assigner":"cve@mitre.org","capec":[],"cvss":4.3,"cvss-time":"2019-05-01T19:29:00","cvss-vector":"AV:N/AC:M/Au:N/C:N/I:N/A:P","cwe":"CWE-476","id":"CVE-2019-11638","impact":{"availability":"PARTIAL","confidentiality":"NONE","integrity":"NONE"},"last-modified":"2019-05-01T19:29:00","references":["https://github.com/TeamSeri0us/pocs/blob/master/recutils/bug-report-recutils","https://github.com/TeamSeri0us/pocs/tree/master/recutils/bug-report-recutils/rec2csv"],"refmap":{"misc":["https://github.com/TeamSeri0us/pocs/blob/master/recutils/bug-report-recutils","https://github.com/TeamSeri0us/pocs/tree/master/recutils/bug-report-recutils/rec2csv"]},"summary":"An issue was discovered in GNU recutils 1.8. There is a NULL pointer dereference in the function rec_field_name_equal_p at rec-field-name.c in librec.a, leading to a crash.","vulnerable_configuration":[{"id":"cpe:2.3:a:gnu:recutils:1.8:*:*:*:*:*:*:*","title":"cpe:2.3:a:gnu:recutils:1.8:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:gnu:recutils:1.8:*:*:*:*:*:*:*"]}